docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,128 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+If your system has the ``curl`` program installed, which is usually true in Linux and Mac,
+.. code-block:: bash
+	
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
+
+.. code-block:: powershell
```

Comment:
Sphinx complains that 
```
.../manim/docs/source/installation/for_dev.rst:19: WARNING: Could not lex literal_block as "powershell". Highlighting skipped.
```

(powershell is an avaliable language for the highlighting though, not sure what is going wrong here.)

Reply:
I also noticed this. Idk what to do.

Reply:
I also got this error way back. Let's just use `bash` or `sh` for now.